server/memphis_handlers_schemas.go

```diff
@@ -118,16 +254,52 @@ func (sh SchemasHandler) GetSchemaVersion(stationVersion int, schemaId primitive
 	return schemaVersion, nil
 }
 
-func (sh SchemasHandler) updateActiveVersion(schemaId primitive.ObjectID, versionNumber int) error {
-	_, err := schemaVersionCollection.UpdateMany(context.TODO(),
-		bson.M{"schema_id": schemaId},
+func (sh SchemasHandler) updateActiveVersion(s *Server, schema models.Schema, versionNumber int) error {
```

Comment:
this function is getting called when the schema gots updated, meand other version has been picked as the active one, but it will apply to the station only when useSchema will be called and the changes will be applied to the station.

That means that no updates needed in here